aspnetcore/grpc/interprocess-namedpipes.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Inter-process communication with gRPC and Named pipes
+author: jamesnk
+description: Learn how to use gRPC for inter-process communication with Named pipes
+monikerRange: '>= aspnetcore-8.0'
+ms.author: jamesnk
+ms.date: 08/08/2022
+uid: grpc/interprocess-namedpipes
+---
+# Inter-process communication with gRPC and Named pipes
+
+By [James Newton-King](https://twitter.com/jamesnk)
+
```

Comment:
It feels like it would be helpful here to have an intro sentence or two and a bullet point or two about when and why to use Named pipes as opposed to gRPC over HTTP.  It could be for your target audience, we don't need an intro, but I'm guessing a fair amount of traffic hitting the topic, might still need to know when this should apply to them.

Reply:
Done. It's basically just a link back to the higher-level doc. Could you take a look?

Reply:
Intro looks good.